tests/annotations/tests.py

```diff
@@ -1016,6 +1016,13 @@ def test_annotation_and_alias_filter_related_in_subquery(self):
                 {"name": "Morgan Kaufmann"},
             ],
         )
+    
+    def test_annotation_exists_empty_query_result(self):
+        Author.objects.create(name='John Smith')
```

Comment:
Tests are failing here since `age` can't be `NULL`, no need to create an author though as you could assert against `Company.objects.annotate(exists=Exists(Company.objects.none()).get().exists()` instead.